app/models/user.rb

```diff
@@ -721,6 +721,17 @@ def get_lat_lon_from_ip
     self.lat_lon_acc_admin_level = lat_lon_acc_admin_level
   end
   
+  def suppressed_emails( suppressed_groups )
```

Comment:
Put a question mark at the end of methods that return a binary. I would rename this `email_suppressed_in_group?` and allow it to take a single group name or an array.